MC/bin/o2dpg_sim_workflow.py

```diff
@@ -519,7 +519,10 @@ def getDPL_global_options(bigshm=False, ccdbbackend=True):
 
 # preproduce the collision context / timeframe structure for all timeframes at once
 precollneeds=[GRP_TASK['name']]
-NEventsQED=10000  # max number of QED events to simulate per timeframe
+# max number of QED events simulated per timeframe.
+# A large pool of QED events (0.6*INTRATE) is needed to avoid repetition of events in the same or
+# neighbouring ITS readout frames, which would fire already activated pixel, discarding the event.
+NEventsQED = 10000 if (INTRATE <= 16668) else int(INTRATE*0.6)
```

Comment:
I think that `NEventsQED = max(10000, int(INTRATE*0.6))` is clearer and avoids putting hard-coded constants. Also, maybe add link to JIRA ticket where this is discussed on more detail.